debug/audio-track-selector.js

```diff
@@ -0,0 +1,114 @@
+(function(videojs) {
```

Comment:
You shouldn't need this file at all. This functionality is included in the latest releases of video.js natively.